webviewer/src/pages/ForgotPassword/components/EmailForm/Form.styles.js

```diff
@@ -0,0 +1,15 @@
+import styled, { css } from 'styled-components';
+import { rem } from 'styles';
+import { rgba } from 'polished';
+import { Form } from 'components/_shared/Form';
+
+export const StyledForm = styled(Form)`
+  max-width: ${({ theme }) => theme.formSize.width};
+`;
+
+export const FormText = styled.p`
+  ${({ theme }) => css`
+    color: ${rgba(theme.colors.black, 0.7)};
+    margin-top: ${rem(-12)};
+  `}
```

Comment:
![localhost_3000_password_forgot](https://user-images.githubusercontent.com/23707448/228520520-dcac411e-996c-4638-8ee8-d5996f44804a.png)
why are you using negative margins? 
whil inspecting layout you can find the problems with markup. please check the demo, screenshot of content-box and fix it.

https://vimeo.com/812768116/cb0a7e7a12